pkg/apis/extensions/types.go

```diff
@@ -87,6 +88,18 @@ type CPUTargetUtilization struct {
 	TargetPercentage int `json:"targetPercentage"`
 }
 
+// Alpha-level support for Custom Metrics in HPA (as annotations).
+type CustomMetricUtilization struct {
```

Comment:
Thats why there is quantity so that float point values can be expressed (as millis).